Mage.Sets/src/mage/cards/a/AlelaCunningConqueror.java

```diff
@@ -81,7 +81,7 @@ public boolean checkTrigger(GameEvent event, Game game) {
         if (!super.checkTrigger(event, game)) {
             return false;
         }
-        Player opponent = game.getPlayer(event.getPlayerId());
+        Player opponent = game.getPlayer(event.getTargetId());
```

Comment:
It's hard to find such replacements from `playerId` to `targetId` due multi-classes inheritance. If you manually checked and replaced all related usages then all fine. If you fix it by failed tests then it's bad.

Fast usages search showing a missing fix like:
`DAMAGED_BATCH_FOR_ONE_PLAYER` -> `OneOrMoreDealDamageTriggeredAbility` -> `DealCombatDamageControlledTriggeredAbility` -> `FelineSovereignTriggeredAbility`:

![shot_240326_080958](https://github.com/magefree/mage/assets/8344157/66b69813-518e-4ed5-b6e2-bbf75358fe5f)

Reply:
good find, I thought I got them all manually but apparently missed that one. multi inheritance certainly complicates that, fortunately since this batch event was recently introduced and not used that many places, it wasn't too bad